rpm/__init__.py

```diff
@@ -39,8 +39,13 @@ def get_sitepackages_and_suffixes(interpreter):
         script = textwrap.dedent(
             """
             import importlib
-            import importlib.machinery
             import json
+            import sys
+            try:
+                import importlib.machinery
+            except ImportError:
+                print(json.dumps({"sitepackages": [], "suffixes": []}))
```

Comment:
oh definitely possible, but i assumed it would be pointless to try python2's rpm module with a python 3.x interpreter. probably no harm in returning the list though